cmd/thanos/sidecar.go

```diff
@@ -137,6 +137,10 @@ func runSidecar(
 					return err
 				}
 
+				level.Info(logger).Log(
+					"msg", "successfully loaded prometheus external labels",
```

Comment:
since this is periodic thing, I would change it to:
```suggestion
					"msg", "successfully updated prometheus external labels",
```

And do it ONLY if the labels are actually changed. That would be the true value.. Thoughts?
Also metric for those would be nice at some point

Reply:
hmm this is onli ontime success on startup isn't it?
the periodical call is in the `runutil.Repeat` below bt maybe I'm missing something out

Reply:
You are right, ignore me. Looked on wrong `m.Update..`